Assets/Scripts/MonthlyTime.cs

```diff
@@ -0,0 +1,34 @@
+using Features.Managers;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class MonthlyTime : MonoBehaviour
+{
+    [SerializeField] private int _daysInAMonth;
+    [SerializeField] private int _secondsInADay;
+
+    private float _secondsPassed;
+    private int _daysPassed;
+    private int _monthsPassed;
+
+    private void FixedUpdate()
+    {
+        _secondsPassed += (1 * GameManager.Instance.GameTimeManager.DeltaTime);
+
+        if (_secondsPassed >= _secondsInADay)
+        {
+            _daysPassed++;
+            _secondsPassed = 0;
+        }
+
+        if(_daysPassed >= _daysInAMonth)
```

Comment:
Do we also have a way to retrieve the current day, month and year?

Reply:
Maybe we can look at returning a name of the month of something when we retrieve the month? Think about customizability from the user. Maybe we want to create custom month names so play around with this. Same for the amount for months.

Reply:
Maybe we don't want months but periods like "Summer", "Fall" etc etc. Could be done the same as a months, just switch the terminology to Periods instead of Months